charts/xp-treatment/ci/ci-values.yaml

```diff
@@ -8,6 +8,8 @@ global:
     useServiceFqdn: true
     uiPrefix: "/xp"
     uiServiceName: xp-management
+  xp-treatment:
+    serviceName: xp-treatment
```

Comment:
Not related to this PR but was it intentional that we decided to use`xp` to represent `xp-management` in the global chart values and everything on the CaraML level? :/ Seems strange to not just call it `xp-management`...

Reply:
I believe it was intentional. If we ignore `xp-treatment`, `xp-management` is just like `merlin`, `turing`, etc. and accessed using `/xp` rather than `/xp-management`.

Reply:
Hmm that makes sense 😅 Never thought of it that way though! Thanks for clarifying!